bee-ternary/src/raw.rs

```diff
@@ -108,7 +108,7 @@ pub trait RawEncodingBuf {
         T::Slice: RawEncoding<Trit = <Self::Slice as RawEncoding>::Trit>,
     {
         // if TypeId::of::<Self>() == TypeId::of::<T>() {
-        //     unsafe { std::mem::transmute(this) }
+        //     unsafe { core::mem::transmute(this) }
```

Comment:
Should we remove this, or explain why we don't use `transmute` for now?

Reply:
Removed